doc-requirements.txt

```diff
@@ -90,7 +90,7 @@ sphinxcontrib-qthelp==1.0.3
     # via sphinx
 sphinxcontrib-serializinghtml==1.1.5
     # via sphinx
-sphinxcontrib-yt==0.2.2
+sphinxcontrib-youtube==1.2.0
```

Comment:
I did this manually. I was not able to do `make doc-requirements.txt` because pip-compile complained:

```
Using legacy resolver. Consider using backtracking resolver with `--resolver=backtracking`.
Could not find a version that matches sphinx<5,<8.0.0,>=0.6,>=1.3,>=1.5.0,>=1.8,>=2,>=7.0.0,~=4.0 (from -r doc-requirements.in (line 2))
Tried: 0.1.61611, 0.1.61798, 0.1.61843, 0.1.61945, 0.1.61950, 0.2, 0.3, 0.4, 0.4.1, 0.4.2, 0.4.3, 0.5, 0.5.1, 0.5.2, 0.6, 0.6.1, 0.6.2, 0.6.3, 0.6.4, 0.6.5, 0.6.6, 0.6.7, 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4, 1.0.5, 1.0.6, 1.0.7, 1.0.8, 1.1, 1.1.1, 1.1.2, 1.1.3, 1.2, 1.2, 1.2, 1.2.1, 1.2.1, 1.2.1, 1.2.2, 1.2.2, 1.2.2, 1.2.3, 1.2.3, 1.3, 1.3, 1.3.1, 1.3.1, 1.3.2, 1.3.2, 1.3.3, 1.3.3, 1.3.4, 1.3.4, 1.3.5, 1.3.5, 1.3.6, 1.3.6, 1.4, 1.4, 1.4.1, 1.4.1, 1.4.2, 1.4.2, 1.4.3, 1.4.3, 1.4.4, 1.4.4, 1.4.5, 1.4.5, 1.4.6, 1.4.6, 1.4.7, 1.4.7, 1.4.8, 1.4.8, 1.4.9, 1.4.9, 1.5, 1.5, 1.5.1, 1.5.1, 1.5.2, 1.5.2, 1.5.3, 1.5.3, 1.5.4, 1.5.4, 1.5.5, 1.5.5, 1.5.6, 1.5.6, 1.6.1, 1.6.1, 1.6.2, 1.6.2, 1.6.3, 1.6.3, 1.6.4, 1.6.4, 1.6.5, 1.6.5, 1.6.6, 1.6.6, 1.6.7, 1.6.7, 1.7.0, 1.7.0, 1.7.1, 1.7.1, 1.7.2, 1.7.2, 1.7.3, 1.7.3, 1.7.4, 1.7.4, 1.7.5, 1.7.5, 1.7.6, 1.7.6, 1.7.7, 1.7.7, 1.7.8, 1.7.8, 1.7.9, 1.7.9, 1.8.0, 1.8.0, 1.8.1, 1.8.1, 1.8.2, 1.8.2, 1.8.3, 1.8.3, 1.8.4, 1.8.4, 1.8.5, 1.8.5, 1.8.6, 1.8.6, 2.0.0, 2.0.0, 2.0.1, 2.0.1, 2.1.0, 2.1.0, 2.1.1, 2.1.1, 2.1.2, 2.1.2, 2.2.0, 2.2.0, 2.2.1, 2.2.1, 2.2.2, 2.2.2, 2.3.0, 2.3.0, 2.3.1, 2.3.1, 2.4.0, 2.4.0, 2.4.1, 2.4.1, 2.4.2, 2.4.2, 2.4.3, 2.4.3, 2.4.4, 2.4.4, 2.4.5, 2.4.5, 3.0.0, 3.0.0, 3.0.1, 3.0.1, 3.0.2, 3.0.2, 3.0.3, 3.0.3, 3.0.4, 3.0.4, 3.1.0, 3.1.0, 3.1.1, 3.1.1, 3.1.2, 3.1.2, 3.2.0, 3.2.0, 3.2.1, 3.2.1, 3.3.0, 3.3.0, 3.3.1, 3.3.1, 3.4.0, 3.4.0, 3.4.1, 3.4.1, 3.4.2, 3.4.2, 3.4.3, 3.4.3, 3.5.0, 3.5.0, 3.5.1, 3.5.1, 3.5.2, 3.5.2, 3.5.3, 3.5.3, 3.5.4, 3.5.4, 4.0.0, 4.0.0, 4.0.1, 4.0.1, 4.0.2, 4.0.2, 4.0.3, 4.0.3, 4.1.0, 4.1.0, 4.1.1, 4.1.1, 4.1.2, 4.1.2, 4.2.0, 4.2.0, 4.3.0, 4.3.0, 4.3.1, 4.3.1, 4.3.2, 4.3.2, 4.4.0, 4.4.0, 4.5.0, 4.5.0, 5.0.0, 5.0.0, 5.0.1, 5.0.1, 5.0.2, 5.0.2, 5.1.0, 5.1.0, 5.1.1, 5.1.1, 5.2.0, 5.2.0, 5.2.0.post0, 5.2.0.post0, 5.2.1, 5.2.1, 5.2.2, 5.2.2, 5.2.3, 5.2.3, 5.3.0, 5.3.0, 6.0.0, 6.0.0, 6.0.1, 6.0.1, 6.1.0, 6.1.0, 6.1.1, 6.1.1, 6.1.2, 6.1.2, 6.1.3, 6.1.3, 6.2.0, 6.2.0, 6.2.1, 6.2.1, 7.0.0, 7.0.0, 7.0.1, 7.0.1
Skipped pre-versions: 0.5.2b1, 0.6b1, 1.0b1, 1.0b2, 1.2b1, 1.2b2, 1.2b3, 1.3b1, 1.3b1, 1.3b2, 1.3b2, 1.3b3, 1.3b3, 1.4a1, 1.4a1, 1.4b1, 1.4b1, 1.5a1, 1.5a1, 1.5a2, 1.5a2, 1.5b1, 1.5b1, 1.6b1, 1.6b1, 1.6b2, 1.6b2, 1.6b3, 1.6b3, 1.7.0b1, 1.7.0b1, 1.7.0b2, 1.7.0b2, 1.8.0b1, 1.8.0b1, 2.0.0b1, 2.0.0b1, 2.0.0b2, 2.0.0b2, 3.0.0b1, 3.0.0b1, 4.0.0b1, 4.0.0b1, 4.0.0b2, 4.0.0b2, 5.0.0b1, 5.0.0b1, 6.0.0b1, 6.0.0b1, 6.0.0b2, 6.0.0b2, 7.0.0rc1, 7.0.0rc1
There are incompatible versions in the resolved dependencies:
  sphinx (from -r doc-requirements.in (line 2))
  Sphinx<8.0.0,>=7.0.0 (from sphinx-prompt==1.7.0->-r doc-requirements.in (line 3))
  Sphinx>=0.6 (from sphinxcontrib-youtube==1.2.0->-r doc-requirements.in (line 8))
  sphinx~=4.0 (from furo==2022.4.7.dev1->-r doc-requirements.in (line 1))
  sphinx<5,>=2 (from sphinx-panels==0.6.0->-r doc-requirements.in (line 9))
  Sphinx>=1.5.0 (from sphinx_fontawesome==0.0.6->-r doc-requirements.in (line 7))
  sphinx>=1.3 (from sphinx-code-include==1.1.1->-r doc-requirements.in (line 5))
  sphinx>=2.0 (from sphinx-material==0.0.35->-r doc-requirements.in (line 4))
  sphinx>=1.8 (from sphinx-copybutton==0.5.2->-r doc-requirements.in (line 6))
make: *** [doc-requirements.txt] Error 2
```
So `sphinx-panels` and `sphinx-prompt` don't agree with each other.

Reply:
`sphinx-panels` is no longer maintained https://github.com/executablebooks/sphinx-panels#sphinx-panels :(